core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/environment/EnvironmentToDetailedEnvironmentV4ResponseConverter.java

```diff
@@ -121,6 +131,32 @@ public DetailedEnvironmentV4Response convert(Environment source) {
         }
         response.setDatalakeResourcesNames(datalakeResourcesNames);
         response.setDatalakeResources(datalakeResourcesResponses);
+        setNetworkIfPossible(response, source);
         return response;
     }
+
+    private void setNetworkIfPossible(DetailedEnvironmentV4Response response, Environment source) {
```

Comment:
Do the same as with the `EnvironmentNetworkValidator`.

Reply:
What you need here is more EnvironmentNetworkV4ResponseConverters that are cloud platform specific.